//23. Write a program to Catch any exception in java. Use try catch and finally statements


package JavaAssignment;

public class Program23A
{
    public static void main(String[] args)
    {
        Program23A obj=new Program23A();
        obj.exceptionHanding();
    }

    public void exceptionHanding()
    {
        int[] arr={1,2,3,4,5,6,7,8};

        try
        {
            int element = arr[9];
            System.out.println("Element at index 3: " + element);
        }
            catch(Exception e)
            {
                System.out.println("Array index out for the Bounds..!!");
            }
        finally
        {
            System.out.println("Finally executed");
        }
    }
}
